upgraderesponder/service.go

```diff
@@ -77,8 +77,16 @@ type Version struct {
 }
 
 type CheckUpgradeRequest struct {
-	AppVersion string            `json:"appVersion"`
-	ExtraInfo  map[string]string `json:"extraInfo"`
+	AppVersion       string            `json:"appVersion"`
+	HarvesterVersion string            `json:"harvesterVersion"`
```

Comment:
Can we add a comment explaining why we need this workaround and when we can remove it?

Reply:
Added it. Thank you.